x-pack/plugins/maps/public/components/metrics_editor/metric_editor.tsx

```diff
@@ -223,6 +233,11 @@ export function MetricEditor({
       {fieldSelect}
       {percentileSelect}
       {labelInput}
+
+      <EuiFormRow label={getMaskI18nLabel({ bucketsName, isJoin })} display="columnCompressed">
+        <MaskExpression fields={fields} isJoin={isJoin} metric={metric} onChange={onChange} />
```

Comment:
nit, doesn't have to block merging this PR.

If we want to ever support logical `OR` operators for multiple metrics that in the future, it might make sense to have a single "Hide grid" field for the `MetricsEditor` rather than per each `MetricEditor`.  It might look something like https://codesandbox.io/s/currying-snowflake-0pz3il?file=/demo.js:0-576. We could do that now and make the `AND` expression unselectable by not having an `onClick` method. Or we could wait to support `OR` operator in the future. In that case the expression could have a popover that allows the user to select either `AND` or `OR`.

Reply:
Interesting idea. Let's wait until there is user requests for supporting `AND` relationship. The issues for this feature have only specified a single mask and have not really specified relationship between masks.